src/main.rs

```diff
@@ -77,7 +121,8 @@ fn main() {
     let socket = bind_socket(local_addr);
     let (sink, stream) = UdpFramed::new(socket, DhtCodec).split();
 
-    let (dht_pk, dht_sk) = gen_keypair();
+    let (dht_pk, dht_sk) = load_or_gen_keys(cli_config.keys_file);
```

Comment:
BTW, AWS requires a server to setup a HDD even for 1 file. So let's try to implement arg `--key` to launch from a secret key only.

Reply:
Add such possibility though - check latest commit. Named parameter as `--secret-key` since just key can be confused with public key.